x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -475,144 +497,144 @@ const ProcessEventDotComponents = React.memo(
-          aria-haspopup={'true'}
-          aria-current={isActiveDescendant ? 'true' : undefined}
-          aria-selected={isSelectedDescendant ? 'true' : undefined}
-          style={nodeViewportStyle}
-          id={nodeId}
-          onClick={handleClick}
-          onFocus={handleFocus}
-          tabIndex={-1}
+      <div
+        data-test-subj={'resolverNode'}
+        className={`${className} kbn-resetFocusState`}
+        role="treeitem"
+        aria-level={adjacentNodeMap.level}
+        aria-flowto={adjacentNodeMap.nextSibling === null ? undefined : adjacentNodeMap.nextSibling}
+        aria-labelledby={labelId}
+        aria-describedby={descriptionId}
+        aria-haspopup={'true'}
+        aria-current={isActiveDescendant ? 'true' : undefined}
+        aria-selected={isSelectedDescendant ? 'true' : undefined}
+        style={nodeViewportStyle}
+        id={nodeId}
+        tabIndex={-1}
+      >
+        <svg
+          viewBox="-15 -15 90 30"
+          preserveAspectRatio="xMidYMid meet"
+          style={{
+            display: 'block',
+            width: '100%',
+            height: '100%',
+            position: 'absolute',
+            top: '0',
+            left: '0',
+          }}
+        >
+          <g>
+            <use
+              xlinkHref={`#${SymbolIds.processCubeActiveBacking}`}
+              fill={backingFill} // Only visible on hover
+              x={-15.35}
+              y={-15.35}
+              stroke={strokeColor}
+              width={markerSize * 1.5}
+              height={markerSize * 1.5}
+              className="backing"
+            />
+            <use
+              role="presentation"
+              xlinkHref={cubeSymbol}
+              x={markerPositionXOffset}
+              y={markerPositionYOffset}
+              width={markerSize}
+              height={markerSize}
+              opacity="1"
+              className="cube"
+            >
+              <animateTransform
+                attributeType="XML"
+                attributeName="transform"
+                type="scale"
+                values="1 1; 1 .83; 1 .8; 1 .83; 1 1"
+                dur="0.2s"
+                repeatCount="1"
+                className="squish"
+                ref={animationTarget}
+              />
+            </use>
+          </g>
+        </svg>
+        <StyledActionsContainer
+          color={colorMap.full}
+          fontSize={scaledTypeSize}
+          topPct={actionableButtonsTopOffset}
         >
-          <svg
-            viewBox="-15 -15 90 30"
-            preserveAspectRatio="xMidYMid meet"
+          <StyledDescriptionText
+            backgroundColor={colorMap.resolverBackground}
+            color={colorMap.descriptionText}
+            isDisplaying={isShowingDescriptionText}
+          >
+            {descriptionText}
+          </StyledDescriptionText>
+          <div
+            className={magFactorX >= 2 ? 'euiButton' : 'euiButton euiButton--small'}
```

Comment:
are these changes intended? If so, should there be a test that uses 'nodeLabel' in this PR?